Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -132,7 +132,7 @@ var DatePickerIOS = React.createClass({
           mode={props.mode}
           minuteInterval={props.minuteInterval}
           timeZoneOffsetInMinutes={props.timeZoneOffsetInMinutes}
-          onChange={this._onChange}
+          onDateChange={this._onChange}
```

Comment:
This is incorrect - the native prop is called onChange. This silences the warning but breaks the functionality (I think).